Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'cocoapods'
 gem 'cocoapods-update-if-you-dare', '~> 0.1.1'
+gem 'colored'
```

Comment:
Yeah that sounds good. I'll open another PR to remove `colored` once https://github.com/Ashton-W/cocoapods-update-if-you-dare/pull/2 is merged and a new version of the gem is released.